### PR TITLE
feat: add `transpile` option (`options.transpile`)

### PR DESCRIPTION
**Pull request by vladflip**
Added transpile option.
http://coffeescript.org/#transpilation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ module.exports = {
 |:--:|:-----:|:----------|
 |**`literate`**|`false`|Enable CoffeeScript in Markdown (Code Blocks) e.g `file.coffee.md`|
 |**`sourceMap`**|`false`|Enable/Disable Sourcemaps|
+|**`transpile`**|`false`|Provide Babel presets and plugins|
```
Comment (michael-ciniawsky) [inline README.md]: Coffeescript uses babel under the hood by default or are other transpilers possible in theory ?

File: README.md [hunk 2]
```diff
@@ -99,6 +100,31 @@ module.exports = {
 }
 ```
 
+### [Transpile](http://coffeescript.org/#transpilation)
```
Comment (michael-ciniawsky) [inline README.md]: Could you do my a favour and wrap all headings in backticks please ?  😛  `\### [Transpile]()` => ```\### [`Transpile`]()```

File: README.md (hunk 1 above)
Comment (vladflip) [reply README.md]: Yeah, there is a built in support only for Babel as I see.

File: README.md (hunk 2 above)
Comment (vladflip) [reply README.md]: Okay.

**Comment by michael-ciniawsky** [comment]
@vladflip Does this feature have any version restrictions e.g `coffeescript >= v2.0.0` etc ? We need to check and eventually guard it then, in case it fails for older versions of CoffeeScript

**Comment by vladflip** [comment]
@michael-ciniawsky It's a new feature yes, just checked with older `coffee-script`, works fine.

**Comment by vladflip** [comment]
@evilebottnawi I don't see any tests in this repo

**Comment by jamesruan** [comment]
hope to see in release quickly
some of my package fail to run because of the lacking of this option

**Comment by vladflip** [comment]
@jamesruan its already there

**Comment by umeboshi2** [comment]
@vladflip It's in the repo, but not quite released.

**Pull request merged**